modules/dhp_connected_devices/app/controllers/dhp_connected_devices/fitbit/fitbit_controller.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require 'fitbit/client'
+
+module DhpConnectedDevices
+  module Fitbit
+
+    class FitbitController < ApplicationController
+      include SentryLogging
+      before_action :feature_enabled
+
+      def connect
+        auth_url = fitbit_api.auth_url_with_pkce
+        redirect_to auth_url
+      end
+
+      def callback
+        begin
+          auth_code = fitbit_api.get_auth_code(callback_params)
+          token_json = fitbit_api.get_token(auth_code)
+          storage_status = token_storage_service.store_tokens(@current_user, 'fitbit', token_json)
+          # todo: remove VeteranDeviceRecordsService after front end reads from local vets-api file
```

Comment:
does this still need to be implemented? or can the comment we removed / captured in a jira ticket?

Reply:
removing this note and adding to JIRA